nexuse2e-webapp/src/main/webapp/WEB-INF/pages/reporting/statistics.jsp

```diff
@@ -24,57 +24,262 @@
+		<c:otherwise>
+			<p class="no-data">no conversations</p>
+		</c:otherwise>
+	</c:choose>
+
+	<h2>Failed Messages</h2>
+	<c:choose>
+	<c:when test="${not empty messages}">
+		<table class="NEXUS_TABLE fixed-table">
+			<colgroup>
+				<col>
+				<col style="width: 6%">
+				<col style="width: 10%">
+				<col style="width: 30%">
+				<col style="width: 30%">
+				<col style="width: 78px">
+			</colgroup>
 			<tr>
-				<td>
-					<nexus:report name="message_stati_24h">
-						<nexus:reportParam name="startDate" value="${last24Hours}"/>
-					</nexus:report>
-				</td>
-				<td>
-					<nexus:report name="conversation_stati_24h">
-						<nexus:reportParam name="startDate" value="${last24Hours}"/>
-					</nexus:report>
-				</td>
+				<th class="NEXUSSection">Time</th>
+				<th class="NEXUSSection">Partner</th>
+				<th class="NEXUSSection">Choreography</th>
+				<th class="NEXUSSection">Conversation</th>
+				<th class="NEXUSSection">Message</th>
+				<th class="NEXUSSection"></th>
 			</tr>
+			<logic:iterate id="message" name="messages">
+				<tr class="${message.messageId}">
+					<td title="${message.createdDate}">
+						${message.createdDate}
+					</td>
+					<td title="${message.partnerId}">
+							${message.partnerId}
+					</td>
+					<td title="${message.choreographyId}">
+						${message.choreographyId}
+					</td>
+					<td title="${message.conversationId}">
+						<nexus:link
+							href="ConversationView.do?convId=${message.nxConversationId}"
+							styleClass="NexusLink">
+							${message.conversationId}
+						</nexus:link>
+					</td>
+					<td title="${message.messageId}">
+						<nexus:link
+							href="MessageView.do?mId=${message.messageId}"
+							styleClass="NexusLink">
+							${message.messageId}
+						</nexus:link>
+					</td>
+					<td><button onClick="this.disabled = true; setContentUrl('ModifyMessage.do?&origin=dashboard&command=requeue&conversationId=${message.conversationId}&messageId=${message.messageId}');">
+						Requeue
+					</button></td>
+				</tr>
+			</logic:iterate>
+		</table>
+		<button onClick="setContentUrl('ProcessConversationReport.do?noReset=true')" class="full-width">Show more</button>
+	</c:when>
+	<c:otherwise>
+		<p class="no-data">no failed messages</p>
+	</c:otherwise>
+	</c:choose>
+
+	<div class="tab">
+		<button class="tablinks" onclick="openTab(event, 'certificates')">Certificates</button>
+		<button class="tablinks" onclick="openTab(event, 'partners')">Partners</button>
+		<button class="tablinks" onclick="openTab(event, 'choreographies')">Choreographies</button>
+	</div>
+
+	<table class="NEXUS_TABLE fixed-table tabcontent" id="partners">
+		<c:choose>
+			<c:when test="${not empty partners}">
+				<colgroup>
+					<col>
+					<col>
+					<col>
+				</colgroup>
+				<tr>
+					<th class="NEXUSSection">Partner ID</th>
+					<th class="NEXUSSection">
+						Last Inbound
+						<div class="info-icon" title="Last successfully sent message (excluding acknowledgements)">?</div>
```

Comment:
Is the title correct? (It is the same as "Last Outbound")

Reply:
But would it be helpful to include this detail (partner or backend) to the title?